src/Microsoft.Health.Dicom.Api/Controllers/WorkitemController.Add.cs

```diff
@@ -46,23 +48,23 @@ public partial class WorkitemController
     [VersionedPartitionRoute(KnownRoutes.AddWorkitemInstancesRoute, Name = KnownRouteNames.PartitionedAddWorkitemInstance)]
     [VersionedRoute(KnownRoutes.AddWorkitemInstancesRoute, Name = KnownRouteNames.AddWorkitemInstance)]
     [AuditEventType(AuditEventSubType.AddWorkitem)]
-    public async Task<IActionResult> AddAsync()
+    public async Task<IActionResult> AddAsync([FromBody] IReadOnlyCollection<DicomDataset> dicomDatasets)
```

Comment:
We can add other annotations here, like `[Required]`, `[MinLength]`, and `[MaxLength]`. I think. Test 'em out. That way we don't have to validate further down the stack

Reply:
Required, Min/Max length are FromRoute specific. I have added FromBody behavior, I hope this is acceptable.